### DISCUSSION TITLE
H3 meetup in San Francisco March 21, 2024

### DISCUSSION THREAD

**Discussion by isaacbrodsky** [uber/h3#814] [General]
We're planning to organize an H3 meetup in San Francisco the evening of March 21, 2024. This would be a chance to meet and talk with H3 developers and other H3 users, over some food and drinks. Please let me know if you're interested! Location to-be-announced in this thread in the coming couple weeks.

**Comment by RichardVasquez** [comment]
South Bay here. Any chance of a Friday or weekend in the future? Thursday evening traffic is rough through the peninsula.

**Reply by isaacbrodsky** [reply]
I'm on the peninsula myself, so happy to have something more accessible to South Bay in the future!

**Reply by marmotzero** [reply]
Also South Bay
